Protocols/CardsDataSource.swift

```diff
@@ -0,0 +1,17 @@
+//
+//  CardsDataSource.swift
+//  Protocols
+//
+//  Created by Felipe Gil on 2021-09-24.
+//
+
+import Foundation
+
+protocol CardsDataSource {
+    var playerName: String? { get set }
+    var cards: [String] { get set }
+    
+    func generateDeckofCards()
+    func getUserName() -> String
```

Comment:
I know this is common in other languages but in Swift we avoid using `get` in methods as they will necessarily return something.

Reply:
Oh, I see. 
Ok, fixed!